render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/solvers/intensity/IntensityMatcher.java

```diff
@@ -0,0 +1,192 @@
+package org.janelia.render.client.newsolver.solvers.intensity;
+
+import ij.process.ColorProcessor;
+import ij.process.FloatProcessor;
+import mpicbg.models.Affine1D;
+import mpicbg.models.Point;
+import mpicbg.models.PointMatch;
+import mpicbg.models.Tile;
+import net.imglib2.util.StopWatch;
+import org.janelia.alignment.spec.TileSpec;
+import org.janelia.alignment.util.ImageProcessorCache;
+import org.janelia.render.client.intensityadjust.intensity.PointMatchFilter;
+import org.janelia.render.client.intensityadjust.intensity.Render;
+import org.janelia.render.client.newsolver.blocksolveparameters.FIBSEMIntensityCorrectionParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.Rectangle;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+class IntensityMatcher {
+	final private PointMatchFilter filter;
+	final private double sameLayerScale;
+	final private double crossLayerScale;
+	final private int numCoefficients;
+	final int meshResolution;
+	final ImageProcessorCache imageProcessorCache;
+
+	public IntensityMatcher(
+			final PointMatchFilter filter,
+			final FIBSEMIntensityCorrectionParameters<?> parameters,
+			final int meshResolution,
+			final ImageProcessorCache imageProcessorCache) {
+		this.filter = filter;
+		this.sameLayerScale = parameters.renderScale();
+		this.crossLayerScale = parameters.crossLayerRenderScale();
+		this.numCoefficients = parameters.numCoefficients();
+		this.meshResolution = meshResolution;
+		this.imageProcessorCache = imageProcessorCache;
+	}
+
+	public void match(final TileSpec p1, final TileSpec p2, final HashMap<String, ArrayList<Tile<? extends Affine1D<?>>>> coefficientTiles) {
+
+		final StopWatch stopWatch = StopWatch.createAndStart();
+		LOG.info("match: entry, pair {} <-> {}", p1.getTileId(), p2.getTileId());
```

Comment:
The logging is pretty verbose here (even for me :).  Since matching typically takes at most a couple of seconds, I suggest removing all of the `LOG.info` calls from this method other than the last one at exit which provides all of the info we really need.